draft-ietf-httpbis-cache-latest.xml

```diff
@@ -1099,7 +1099,17 @@
 <t>
    Because unsafe request methods (<xref target="safe.methods"/>) such as PUT, POST or DELETE
    have the potential for changing state on the origin server, intervening
-   caches can use them to keep their contents up to date.
+   caches are required to invalidate stored responses to keep their contents up to date.
+   <x:dfn>Invalidate</x:dfn> means that the cache will either remove all
```

Comment:
Not sure about the change from "can use" to "are required".

Are we making a normative change here?

Reply:
The requirement is already stated below; this is not normative, just aligning the prose with the requirements.